frappe/public/css/website.css

```diff
@@ -410,6 +410,7 @@ a.no-decoration:active {
 .web-footer {
   padding: 20px 0px;
   min-height: 140px;
+  border-top: 1px solid #ebeff2;
 }
```

Comment:
standard border color is `#d1d8dd`. You should update the `.less` file not `.css`